### PR TITLE
various updates

### PR DESCRIPTION
**Pull request by vesper2000**
- additional content
- demo cleanup
- readme cleanup
- removal of icons
  #patch#

### CODE REVIEW

File: addon/components/frost-button.js [hunk 1]
```diff
@@ -71,6 +71,12 @@ function addDesignClass (design, classes) {
     case 'tab':
       classes.push('tab')
       break
+    case 'info-bar':
+      classes.push('info-bar')
+      break
+    case 'in-line':
```
Comment (sglanzer-fire) [inline addon/components/frost-button.js]: Let's drop this so that nobody gets confused

File: addon/styles/_frost-button.scss [hunk 2]
```diff
@@ -129,10 +131,14 @@ $frost-color-button-tertiary-active-bg: #e7eef3 !global;
     }
   }
 
+  // ----------------------
   // Sizes
+  // ----------------------
+
+  // size = 'extra-large'
```
Comment (sglanzer-fire) [inline addon/styles/_frost-button.scss:138]: Did Brian confirm if extra-large is still a thing?

File: addon/styles/_frost-button.scss [hunk 3]
```diff
@@ -163,10 +171,11 @@ $frost-color-button-tertiary-active-bg: #e7eef3 !global;
     }
   }
 
+  // size = 'size'
```
Comment (sglanzer-fire) [inline addon/styles/_frost-button.scss]: Minor - size = 'small'

File: addon/styles/_frost-link.scss [hunk 4]
```diff
@@ -109,6 +127,12 @@ $frost-color-link-action-active:  rgba($frost-color-blue-1, 0.1);
         .content {
           opacity: 0.5;
         }
+
+        .text{
+          border-bottom-width: 1px;
+          border-bottom-style: solid;
+          border-bottom-color: $frost-color-blue-1;
```
Comment (sglanzer-fire) [inline addon/styles/_frost-link.scss:134]: Please raise an issue to convert these to aliases

File: addon/templates/components/frost-button.hbs [hunk 5]
```diff
@@ -1,24 +1,36 @@
...
-{{/if}}
-{{#if isIconOnly}}
-  <div class="icon">
-    {{frost-icon icon=icon}}
+{{#if (eq design 'info-bar')}}
+  <div class='content'>
+    <div class=' text'>
+      <div class='svg'>{{frost-icon icon=icon}}</div>
+      <div class="text">{{text}}</div>
+      {{yield}}
```
Comment (sglanzer-fire) [inline addon/templates/components/frost-button.hbs]: What is the yield for here?

File: public/svgs/frost/infobar-create.svg [hunk 6]
```diff
@@ -1,6 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (sglanzer-fire) [inline public/svgs/frost/infobar-create.svg:1]: I think we should just have 'plus' and 'find' icons and just scale them to the scenarios that require those.  Please raise an issue to work on this.

File: public/svgs/frost/round-add.svg [hunk 7]
```diff
@@ -1,6 +1,10 @@
-<svg viewBox="0 0 20 20">
-	<rect x="9.5" y="4" fill="#009EEF" width="1" height="12"/>
-	<rect x="4" y="9.5" fill="#009EEF" width="12" height="1"/>
-	<path fill="#009EEF" d="M10,1c5,0,9,4,9,9s-4,9-9,9s-9-4-9-9S5,1,10,1z M10,17.9c4.3,0,7.9-3.5,7.9-7.9c0-4.3-3.5-7.9-7.9-7.9
-	S2.1,5.7,2.1,10C2.1,14.3,5.7,17.9,10,17.9z"/>
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (sglanzer-fire) [inline public/svgs/frost/round-add.svg:1]: This gets into the composite svg concept - basically we want a round svg that we can use as a backing for any other svg.  If we don't already have an issue to look at this, please raise one.

File: addon/styles/_frost-button.scss (hunk 2 above)
Comment (vesper2000) [inline addon/styles/_frost-button.scss:138]: At the moment it is not being used.  He wants to keep it though. I kept the code, but did not expose this in the readme and the demo.

File: addon/templates/components/frost-button.hbs (hunk 5 above)
Comment (vesper2000) [inline addon/templates/components/frost-button.hbs]: Not needed.

**Comment by sglanzer-fire** [comment]
Approved

[Approved with PullApprove](https://pullapprove.com/ciena-frost/ember-frost-core/pull-request/29/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ciena-frost/ember-frost-core)

**Pull request merged**